FishBun/src/main/java/com/sangcomz/fishbun/permission/PermissionCheck.java

```diff
@@ -25,6 +26,27 @@ public PermissionCheck(Context context) {
         this.context = context;
     }
 
+    public boolean isCameraPermissionDeclared() {
+
+        boolean isCameraPermissionDeclared = false;
+
+        try {
+            PackageInfo info = context.getPackageManager().getPackageInfo(context.getPackageName(), PackageManager.GET_PERMISSIONS);
+            String[] permissions = info.requestedPermissions;//This array contains the requested permissions.
+
+            if (permissions != null && permissions.length > 0) {
+                for (String permission : permissions) {
+                    if (permission.equals("android.permission.CAMERA")) {
```

Comment:
It is better to use declared constants.
 "android.permission.CAMERA"  ->`Manifest.permission.CAMERA`

Reply:
> It is better to use declared constants.
> "android.permission.CAMERA" ->`Manifest.permission.CAMERA`

Sorry i forgot that there is a constant for permission. i did stackoverflow so there it was mentioned in string format. This is a small change